### ISSUE TITLE
Bdv-playground dependencies

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#504]
@NicoKiaru

Currently we are having this:

```
		<!-- Java 8	Update Site	-->
		<imglib2.version>5.11.1</imglib2.version>
		<imglib2-cache.version>1.0.0-beta-16</imglib2-cache.version>
		<bigdataviewer-core.version>10.2.1</bigdataviewer-core.version>
		<bigdataviewer-vistools.version>1.0.0-beta-28</bigdataviewer-vistools.version>

		<!-- BIOP -->
		<bigdataviewer-playground.version>0.3.8-SNAPSHOT</bigdataviewer-playground.version>
```

and 

```
<parent>
		<groupId>org.scijava</groupId>
		<artifactId>pom-scijava</artifactId>
		<version>30.0.0</version>
	</parent>
```

What would you recommend using?

**Comment by NicoKiaru** [comment]
bdv-playground: 0.3.13 (there's probably not much change into 0.3.13
imglib2 : 5.12.0 is available ??? Maven shows 5.10.0
imglib2 cache : ok
bdv-core : ok
bdv-vistools : ok

**Comment by tischi** [comment]
> imglib2 : 5.12.0 is available ??? Maven shows 5.10.0

yes, I think that's the one shipped with Fiji...

**Comment by NicoKiaru** [comment]
Right, I was confused beacause scijava maven shows it, but it's the 'most popular', not the most recent.
